app-text/glogg/glogg-1.1.4.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="The fast, smart log explorer"
+DESCRIPTION="glogg is a multi-platform GUI application to browse and search \
+through long or complex log files. It is designed with programmers and system \
+administrators in mind. glogg can be seen as a graphical, interactive \
+combination of grep and less."
+HOMEPAGE="http://glogg.bonnefon.org"
+COPYRIGHT="2009-2017 Nicolas Bonnefon and other contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/nickbnf/glogg/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="1226b726317f42a52caa708852d40f59665dbc7d96e2520aff7640c373c67ce8"
+SOURCE_FILENAME="glogg.$portVersion.tar.gz"
+SOURCE_DIR="glogg-$portVersion"
+ADDITIONAL_FILES="glogg.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qview$secondaryArchSuffix = $portVersion
```

Comment:
I think you meant glogg here instead of qview?
```diff
- 	qview$secondaryArchSuffix = $portVersion
+ 	glogg$secondaryArchSuffix = $portVersion
```